kbase-extension/static/kbase/js/widgets/kbaseTabs.js

```diff
@@ -241,47 +282,72 @@ define(['kbwidget', 'bootstrap', 'jquery', 'kbaseDeletePrompt'], (
             return this.data('tabs')[tabName];
         },
 
-        showTab: function (tab) {
-            this.data('nav')[tab].find('a').trigger('click');
+        showTab: function (tabName) {
+            this.data('nav')[tabName].find('a').trigger('click');
         },
 
         removeTab: function (tabName) {
             const $tab = this.data('tabs')[tabName];
             const $nav = this.data('nav')[tabName];
+            const deleteNav = () => {
```

Comment:
extracted the deletion tab nav and panel as they are re-used, and also used separately.